crabs/detection_tracking/evaluate_model.py

```diff
@@ -0,0 +1,161 @@
+import argparse
+import json
+import os
+
+import torch
+from detection_utils import (
+    create_dataloader,
+    get_test_transform,
+    myFasterRCNNDataset,
+)
+from evaluate import test_detection
+
+# select device (whether GPU or CPU)
+device = (
+    torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
+)
+
+
+class Detector_Test:
```

Comment:
same comment about using test in classes and functions. I'm not sure if pytest would be confused in this case but I still find it confusing to have the word test in something that is not an actual test?

Reply:
maybe "model evaluation" or "detector evaluation"? I find I cannot tell very well what this class does from its name

Reply:
Yes, Sorry about that. the code used to be train and test. I changed some that failed the pytest. Will change them all to be consistent.